core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerAPI.java

```diff
@@ -346,19 +364,33 @@ public void suspendTable(final CharSequence tableName) {
         }
     }
 
-    // Check if sequencer files exist, e.g. is it WAL table sequencer must exist
-    private static boolean isWalTable(final CharSequence tableName, final CharSequence root, final FilesFacade ff) {
-        Path path = Path.getThreadLocal2(root);
-        return isWalTable(tableName, path, ff);
-    }
+//    // Check if sequencer files exist, e.g. is it WAL table sequencer must exist
```

Comment:
Looks like a leftover.

Reply:
done in [420e967](https://github.com/questdb/questdb/pull/2752/commits/420e967ac07a79d0b56018aae7947c697cbcd88b)